.github/workflows/dev-new.yaml

```
@@ -0,0 +1,55 @@
+    branches:
+      - lais-1.0
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
```

actionlint (1.7.4)

the runner of "docker/login-action@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)